### PR TITLE
Rename all uses of GTU to CCD.

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

Changes following the renaming of GTU token to CCD.

## Changes

- rename GTU token to CCD
- rename send-gtu, send-gtu-scheduled and send-gtu-encrypted to send,
  send-scheduled and send-shielded.

## Checklist

- [ ] My code follows the style of this project.
- [ ] The code compiles without warnings.
- [ ] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: src/Concordium/Client/Output.hs [hunk 1]
```diff
@@ -306,7 +306,7 @@ printContractInfo CI.ContractInfo{..} namedOwner namedModRef = do
   tell [ [i|Contract:        #{contractName}|]
        , [i|Owner:           #{owner}|]
        , [i|ModuleReference: #{showNamedModuleRef namedModRef}|]
-       , [i|Balance:         #{Types.amountToString ciAmount} GTU|]
+       , [i|Balance:         #{Types.amountToString ciAmount} CCD|]
```
Comment (Bargsteen) [inline src/Concordium/Client/Output.hs:309]:
```suggestion
       , [i|Balance:         #{showCcd ciAmount}|]
```

File: src/Concordium/Client/Runner.hs [hunk 2]
```diff
@@ -1200,7 +1200,7 @@ encryptedTransferTransactionConfirm EncryptedTransferTransactionConfig{..} confi
         = ettTransactionCfg
 
   logInfo
-    [ printf "transferring %s GTU from encrypted balance of account %s to %s" (Types.amountToString ettAmount) (showNamedAddress addr) (showNamedAddress ettReceiver)
+    [ printf "transferring %s CCD from encrypted balance of account %s to %s" (Types.amountToString ettAmount) (showNamedAddress addr) (showNamedAddress ettReceiver)
```
Comment (Bargsteen) [inline src/Concordium/Client/Runner.hs:1203]:
```suggestion
    [ printf "transferring %s CCD from shielded balance of account %s to %s" (Types.amountToString ettAmount) (showNamedAddress addr) (showNamedAddress ettReceiver)
```

Also, we could use `showCcd` consistently here and several places below.

File: src/Concordium/Client/Runner.hs [hunk 3]
```diff
@@ -1383,7 +1383,7 @@ accountEncryptTransactionConfirm AccountEncryptTransactionConfig{..} confirm = d
 
 
   logInfo $
-    [ printf "transferring %s GTU from public to encrypted balance of account %s" (Types.amountToString aeAmount) (showNamedAddress addr)
+    [ printf "transferring %s CCD from public to encrypted balance of account %s" (Types.amountToString aeAmount) (showNamedAddress addr)
```
Comment (Bargsteen) [inline src/Concordium/Client/Runner.hs:1386]:
```suggestion
    [ printf "transferring %s CCD from public to shielded balance of account %s" (Types.amountToString aeAmount) (showNamedAddress addr)
```

File: src/Concordium/Client/Runner.hs [hunk 4]
```diff
@@ -1401,7 +1401,7 @@ accountDecryptTransactionConfirm AccountDecryptTransactionConfig{..} confirm = d
         = adTransactionCfg
 
   logInfo $
-    [ printf "transferring %s GTU from encrypted to public balance of account %s" (Types.amountToString (Enc.stpatdTransferAmount adTransferData)) (showNamedAddress addr)
+    [ printf "transferring %s CCD from encrypted to public balance of account %s" (Types.amountToString (Enc.stpatdTransferAmount adTransferData)) (showNamedAddress addr)
```
Comment (Bargsteen) [inline src/Concordium/Client/Runner.hs:1404]:
```suggestion
    [ printf "transferring %s CCD from shielded to public balance of account %s" (Types.amountToString (Enc.stpatdTransferAmount adTransferData)) (showNamedAddress addr)
```

File: src/Concordium/Client/Commands.hs [hunk 5]
```diff
@@ -729,21 +728,21 @@ accountListCmd =
...
 accountEncryptCmd =
   command
-    "encrypt"
+    "shield"
     (info
       (AccountEncrypt <$>
         transactionOptsParser <*>
-        option (eitherReader amountFromStringInform) (long "amount" <> metavar "GTU-AMOUNT" <> help "The amount to transfer to encrypted balance."))
+        option (eitherReader amountFromStringInform) (long "amount" <> metavar "CCD-AMOUNT" <> help "The amount to transfer to encrypted balance."))
       (progDesc "Transfer an amount from public to encrypted balance of the account."))
```
Comment (Bargsteen) [inline src/Concordium/Client/Commands.hs:736]:
```suggestion
      (progDesc "Transfer an amount from public to shielded balance of the account."))
```

File: src/Concordium/Client/Runner.hs (hunk 2 above)
Comment (abizjak) [reply src/Concordium/Client/Runner.hs:1203]: One thing at a time. :)

**Review by Bargsteen — APPROVED** [review]

**Pull request merged**